### ISSUE TITLE
Remove hard dependency on environment variables

### ISSUE THREAD

**Issue by pmelab** [amazeeio-solutions/silverback-mono#726]
We are relying on environment variables to be set and therefore use `dotenv` and `source .envrc` extensively in npm scripts throughout the monorepo. Thats why we ended up prepending most (but not all!) scripts with `source .envrc` which is only available in `/bin/bash`, which is not the default shell everywhere, which is why we are overriding the `yarn` `script-shell` with `bash` in some places, but it breaks in others.

In general, tests and development should work without any environment variables being set and `dotenv` is a convenient way to locally override environments.

Goals:

- [ ] all occurences of `source .envrc` are gone
- [ ] all occurences of `yarn config set script-shell /bin/bash` are gone
- [ ] make `amazeelabs/silverback-cli` detect a non-hosting environment without relying on environment variables (database credentials?)
- [ ] all tests still pass

**Comment by Leksat** [comment]
I would also get rid if `direnv`:
- instead of extending `$PATH`, we can create yarn scripts
- instead of automatic node version switch with NVM, we can use `engines.node` in `package.json`

General ideas:
- have as less dot-files as possible
- have as less requirements as possible
- make everything very explicit
- use yarn for everything

**Comment by pmelab** [comment]
I agree to all of your points. But I was more referring to automatic loading of `.env`. Some places in code rely on environment variables being set. They should just work with sensible defaults.

**Comment by Leksat** [comment]
Currently `.env` files are gitignored and generated by `direnv`. This causes issues on updating env vars - other devs still have the outdated `.env` file.

We could either commit `.env` file, or add env vars directly to yarn scripts.
